components/bl-collapsiblePanel-component/src/index.js

```diff
@@ -0,0 +1,73 @@
+import { useMemo, useState } from 'react';
+
+const { cn } = BackendlessUI.CSSUtils;
+
+export default function CollapsiblePanelComponent({ component, elRef, eventHandlers, pods }) {
+  const { classList, style, display, title, activeTitle } = component;
+  const { onOpen, onClose } = eventHandlers;
+
+  const [isActive, setIsActive] = useState(false);
+  const [podClass, setPodClass] = useState('');
+
+  const changeActive = currentIsActive => {
+    if(currentIsActive) {
+      setIsActive(false);
+      setPodClass('close');
```

Comment:
`<div className={ cn('panel-content', isActive ? 'open'  : 'close') }>`
https://github.com/Backendless/ui-builder-library/pull/276#discussion_r1113492325

Reply:
We cannot give a class on the first render(due to the use of animation in styles), it is only assigned when the IsActive changes onClick event.